packages/useWorker/package.json

```
@@ -15,7 +15,7 @@
     "test": "npm run build && npm run test:unit"
   },
   "peerDependencies": {
-    "react": "^16.8.0"
+    "react": ">=16.0.0 <20.0.0"
```

```suggestion
    "react": ">=16.8.0 <20.0.0"
```

ref: https://legacy.reactjs.org/docs/hooks-reference.html